src/Slices/ServiceCatalog/UI/Page.test.tsx

```diff
@@ -100,6 +100,7 @@ test("ServiceCatalog shows updated services", async () => {
   expect(
     await screen.findByRole("generic", { name: "ServiceCatalog-Empty" })
   ).toBeInTheDocument();
+  expect(await screen.findByText("Update Service Catalog")).toBeInTheDocument();
```

Comment:
it can be changed here and other tests cases to words() function as that button use it